.github/workflows/lint-pr-adr.yml

```diff
@@ -0,0 +1,120 @@
+name: PR ADR Validation Workflow
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
+
+jobs:
+  validate_adr_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Parse PR label and title
+        uses: actions/github-script@v6
+        id: lint_title_adr
+        with:
+          result-encoding: json
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const title = context.payload.pull_request.title;
+            const labels = context.payload.pull_request.labels;
+            const hasAdrLabel = labels.some(label => label.name === 'kind/adr');
+            const re = /^chore:\s*\[ADR-(\d{4})\]/;
```

Comment:
@KhudaDad414 I see your point and make sense, especially if we are planning to drop semantic release titles. However as the PR ADRs have no intent to trigger any release I'm wondering if we couldn't just create a new `ADR` repo for the AsyncAPI initiative and invite other projects to use it as well.

Any thoughts folks ?

Reply:
@KhudaDad414 what if instead of blocking this PR we make the regex to either support `chore: [ADR-NNNN]` or `[ADR-NNNN]` as the title? This way we can already unblock this, start using ADRs, and as soon as the migration to changesets is done, we can remove the `chore: [ADR-NNNN]` option.

Reply:
I've seen this in the [doc](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md#some-handy-advice)
> Since changesets are focused on releases and changelogs, changes to your repository that don't require these won't need a changeset. As such, we recommend not adding a blocking element to contributions in the absence of a changeset.

It means that contributors will only commit the ADR files without the need to bump a release or use changesets ?

Reply:
Yeah, IMHO ADRs should not trigger a new release and, therefore, it shouldn't contain a changeset.

Reply:
@fmvilas agreed. let's merge this one.